src/platform/qt/ArchiveInspector.cpp

```diff
@@ -13,10 +13,11 @@ ArchiveInspector::ArchiveInspector(const QString& filename, QWidget* parent)
 	: QDialog(parent)
 {
 	m_ui.setupUi(this);
-	connect(m_ui.archiveView, &LibraryView::doneLoading, [this]() {
-		m_ui.loading->hide();
-	});
-	connect(m_ui.archiveView, SIGNAL(accepted()), this, SIGNAL(accepted()));
+	//connect(m_ui.archiveView, &LibraryController::doneLoading, [this]() {
+	//	m_ui.loading->hide();
+	//});
+	connect(m_ui.archiveView, &LibraryController::startGame, this, &ArchiveInspector::accepted);
```

Comment:
So, about that. The SIGNAL/SLOT notation is stringly typed, which means it'll compile no matter what happens, and if the signals/slots don't exist you'll get only a warning printed to the command line (which I think I saw some of... might need to look into that.)

If you use the `&method` syntax (C++11 and up only), it does type and function checking at compile time, so if the functions are incompatible or don't exist at all, it'll fail to compile. (Plus it looks cleaner imo.)

So for now it doesn't match the rest of the code, but it would make more sense to make the rest of the code match this rather than the other way round.